[2025-03-14T16:01:29.205+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gen_data_daily.run_script scheduled__2025-03-10T22:00:00+00:00 [queued]>
[2025-03-14T16:01:29.219+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gen_data_daily.run_script scheduled__2025-03-10T22:00:00+00:00 [queued]>
[2025-03-14T16:01:29.220+0000] {taskinstance.py:2171} INFO - Starting attempt 8 of 9
[2025-03-14T16:01:29.254+0000] {taskinstance.py:2192} INFO - Executing <Task(BashOperator): run_script> on 2025-03-10 22:00:00+00:00
[2025-03-14T16:01:29.264+0000] {standard_task_runner.py:60} INFO - Started process 670 to run task
[2025-03-14T16:01:29.275+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'gen_data_daily', 'run_script', 'scheduled__2025-03-10T22:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/gen_data_daily.py', '--cfg-path', '/tmp/tmp4dbrw382']
[2025-03-14T16:01:29.276+0000] {standard_task_runner.py:88} INFO - Job 11: Subtask run_script
[2025-03-14T16:01:29.375+0000] {task_command.py:423} INFO - Running <TaskInstance: gen_data_daily.run_script scheduled__2025-03-10T22:00:00+00:00 [running]> on host 91a15d92d4ed
[2025-03-14T16:01:29.521+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='longnv' AIRFLOW_CTX_DAG_ID='gen_data_daily' AIRFLOW_CTX_TASK_ID='run_script' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T22:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='8' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T22:00:00+00:00'
[2025-03-14T16:01:29.522+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-14T16:01:29.523+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '\n        python /opt/***/external_scripts/gen_transaction_data.py 2025-03-11\n        ']
[2025-03-14T16:01:29.541+0000] {subprocess.py:86} INFO - Output:
[2025-03-14T16:01:30.170+0000] {subprocess.py:93} INFO - Data retrieved! (20000, 6)
[2025-03-14T16:01:30.172+0000] {subprocess.py:93} INFO - Data saved! /tmp/***tmp7vwcm0vv
[2025-03-14T16:01:30.238+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-14T16:01:30.293+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=gen_data_daily, task_id=run_script, execution_date=20250310T220000, start_date=20250314T160129, end_date=20250314T160130
[2025-03-14T16:01:30.350+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-14T16:01:30.400+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
